server/src/controllers/parcels/changeStatus.ts

```diff
@@ -0,0 +1,13 @@
+import { changeStatusQuery } from '../../database/queries';
+
+const changeStatus = async (req:any, res:any) => {
+  const { userId: sellerId } = req;
+  const { id } = req.params;
```

Comment:
I suppose you should change the name here, instead of id you can take a Parcel Id, this is just to be more readable.